lib/mainloop-control.c

```diff
@@ -34,37 +34,50 @@
 
 #include <string.h>
 
+static gboolean
+_control_process_compat_log_command(const gchar *level, const gchar *onoff, GString *result)
+{
+  if (!level)
+    return FALSE;
+
+  gint ll = msg_map_string_to_log_level(level);
+  if (ll < 0)
+    return FALSE;
+
+  if (onoff)
+    {
+      gboolean on = g_str_equal(onoff, "ON");
+      if (!on)
+        ll = ll - 1;
```

Comment:
One last note:

Asking our users to enable debug logging in production for a short period of time is quite usual, the support team does this frequently in the case of syslog-ng PE.

"Debug on" and then "debug off" now are not inverse functions.

I think this can be documented and we can inform the affected parties about the change, and the new intuitive log-level alternative.